rex/resource.py

```diff
@@ -480,7 +480,22 @@ def _extract_ds_slice(self, ds_slice):
             if ax_idx is not None:
                 idx_slice += (ax_idx,)
 
-        out = self.ds[slices]
+        try:
+            out = self.ds[slices]
+        except Exception as e:
+            msg = (f'Error retrieving h5pyd data from "{self.ds}" for '
```

Comment:
Very minor but isn't it more correct to have a  message like this
```suggestion
            msg = (f'Error retrieving data from "{self.ds}" for '
```
since it's not necessarily an `h5pyd` error at this point?